server/lib/house/house.update.js

```diff
@@ -24,6 +25,8 @@ async function update(selector, newHouse) {
 
   await house.update(newHouse);
 
+  this.event.emit(EVENTS.HOUSE.UPDATE);
```

Comment:
maybe we want to send the newHouse here?

```
this.event.emit(EVENTS.HOUSE.UPDATED, house.get({ plain: true}));
```

Even if it's not useful in this PR, I think it make sense for future uses